contracts/governed_token_wrapper/lib.rs

```diff
@@ -729,7 +729,7 @@ pub mod governed_token_wrapper {
         pub fn get_fee_from_amount(&mut self, amount_to_wrap: Balance) -> Balance {
             amount_to_wrap
                 .saturating_mul(self.fee_percentage)
-                .saturating_div(100)
+                .saturating_div(10000)
```

Comment:
I think it is better to get rid of these numbers, set the constant and use the constant value everywhere.